examples/employee-app/LICENSE.md

```diff
@@ -0,0 +1,47 @@
+# Oracle Employee Directory Sample Application

```

Comment:
Please remove this file, Helidon is licensed under apache v2.

Reply:
Removed the file.